client/src/ui/organisms/sidebar/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import "./index.scss";
+
+export function RenderSideBar({ doc }) {
+  console.log(doc);
+  if (!doc.related_content) {
+    if (doc.sidebarHTML) {
+      return (
+        <aside
+          id="sidebar-quicklinks"
+          className="sidebar"
+          dangerouslySetInnerHTML={{
+            __html: `<h4>Related Topics</h4>${doc.sidebarHTML}`,
```

Comment:
👍 I was honestly not sure how best to do it so, I was hoping to get your feedback 😸

Reply:
I am not sure there is a way to avoid this. I am not sure if there is another way to convert a plain string to HTML in React. I say that because if there was, you could probably do something like:

```
return (
  <SidebarContainer>
    <h4>Related Topics</h4>
    { React.toHTML(doc.sidebarHTML) }
  </SidebarContainer>
);
```